workflow-testing/src/main/java/com/squareup/workflow1/testing/RenderTester.kt

```diff
@@ -237,7 +237,7 @@ interface RenderTester<PropsT, StateT, OutputT, RenderingT> {
    * expectation by returning a [ChildWorkflowMatch]. If the expectation matches, the function
    * must include the rendering and optional output for the child workflow.
    */
-  fun expectWorkflow(
+  internal abstract fun expectWorkflow(
```

Comment:
We don't expect there to be any public callers of this method, so just hiding it completely. We can always make it public later.